webapp/src/language/en.json

```diff
@@ -86,7 +86,63 @@
+    "mainnet/cpu-benchmark>title": "CPU Benchmarks  - EOS Mainnet Network Dashboard",
+    "mainnet/block-producers>title": "Block Producers - EOS Mainnet Network Dashboard",
+    "mainnet/rewards-distribution>title": "Rewards Distribution - EOS Mainnet Network Dashboard",
+    "mainnet/nodes>title": "EOS Mainnet Block Producer Nodes Status Monitor",
+    "mainnet/nodes-distribution>title": "Geographic Distribution of Nodes - EOS Mainnet Network Dashboard",
+    "mainnet/accounts>title": "Accounts and Contracts - EOS Mainnet Network Dashboard",
+    "mainnet/bpjson>title": "BP JSON - EOS Mainnet Network Dashboard",
+    "mainnet/ricardian-contract>title": "BP Agreement - EOS Mainnet Network Dashboard",
+    "mainnet/block-distribution>title": "Block Production Distribution - EOS Mainnet Network Dashboard",
+    "mainnet/missed-blocks>title": "Missed Blocks - EOS Mainnet Network Dashboard",
+    "mainnet/endpoints>title": "API Endpoints - EOS Mainnet Network Dashboard",
+    "mainnet/about>title": "About - EOS Mainnet Network Dashboard",
+    "mainnet/help>title": "Help - EOS Mainnet Network Dashboard",
+    "telos": "Telos Testnet Network Dashboard and Infrastructure Monitor",
+    "telos/cpu-benchmark>title": "CPU Benchmarks  - Telos Testnet Network Dashboard",
+    "telos/block-producers>title": "Block Producers - Telos Testnet Network Dashboard",
+    "telos/rewards-distribution>title": "Rewards Distribution - Telos Testnet Network Dashboard",
+    "telos/nodes>title": "Telos Testnet Nodes Status Monitor",
+    "telos/nodes-distribution>title": "Geographic Distribution of Nodes - Telos Testnet Network Dashboard",
+    "telos/accounts>title": "Accounts and Contracts - Telos Testnet Network Dashboard",
+    "telos/bpjson>title": "BP JSON - Telos Testnet Network Dashboard",
+    "telos/ricardian-contract>title": "BP Agreement - Telos Testnet Network Dashboard",
+    "telos/block-distribution>title": "Block Production Distribution - Telos Testnet Network Dashboard",
+    "telos/missed-blocks>title": "Missed Blocks - Telos Testnet Network Dashboard",
+    "telos/endpoints>title": "API Endpoints - Telos Testnet Network Dashboard",
+    "telos/about>title": "About - Telos Testnet Network Dashboard",
+    "telos/help>title": "Help - Telos Testnet Network Dashboard",
+    "jungle": "Jungle Testnet Network Dashboard and Infrastructure Monitor",
+    "jungle/cpu-benchmark>title": "CPU Benchmarks  - Jungle Testnet Network Dashboard",
+    "jungle/block-producers>title": "Block Producers - Jungle Testnet Network Dashboard",
+    "jungle/rewards-distribution>title": "Rewards Distribution - Jungle Testnet Network Dashboard",
+    "jungle/nodes>title": "Jungle Testnet Block Producer Nodes Status Monitor",
+    "jungle/nodes-distribution>title": "Geographic Distribution of Nodes - Jungle Testnet Network Dashboard",
+    "jungle/accounts>title": "Accounts and Contracts - Jungle Testnet Network Dashboard",
```

Comment:
@tetogomez we can have a translation file for each network and in that file override any translation, so we should have en.telos.json and en.jungle.json and there override the "accounts>title": "value" without need to use the network name in the route